Packs/GoogleMaps/Integrations/GoogleMaps/GoogleMaps.py

```
@@ -0,0 +1,126 @@
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+# Disable insecure warnings
+OUTPUTS_PREFIX = 'GoogleMaps'
+NO_RESULTS_MESSAGE = 'No matching places were found.'
+
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+class Client(BaseClient):
+    def __init__(self, api_key: str, base_url: str, proxy: bool, insecure: bool):
+        """
+        Client to use in the IPinfo integration. Uses BaseClient
+        """
+        super().__init__(base_url=base_url,
+                         verify=not insecure,
+                         proxy=proxy)
+        self.api_key = api_key
+
+    def google_maps_geocode(self, search_address: str, error_on_no_results: bool) -> List[CommandResults]:
+        # noinspection PyTypeChecker
+        response: Dict[str, Any] = self._http_request(method='GET', url_suffix='geocode/json?',
+                                                      params=assign_params(address=search_address, key=self.api_key))
+
+        status = demisto.get(response, 'status')
+        if status == 'OK':
+            return parse_response(response, search_address)
+
+        elif status == 'ZERO_RESULTS':
+            if error_on_no_results:
+                raise DemistoException(message=NO_RESULTS_MESSAGE, res=response)
+
+            return [CommandResults(readable_output=NO_RESULTS_MESSAGE)]
```

Please return the error itself, and create the `CommandResults` in the `_command` function.

Also please note it's fine to return the result wrapped in a list, but a single `CommandResults` is even better since `return_results` can handle both cases, and is more efficient without the wrapper list.